examples/opentype_variations.py

```diff
@@ -0,0 +1,66 @@
+from pathlib import Path
+from string import ascii_lowercase, ascii_uppercase
+
+from matplotlib import font_manager as fm, pyplot as plt
+from matplotlib.widgets import RadioButtons, Slider
+
+
+if not Path(__file__).with_name('Hubot-Sans.ttf').exists():
```

Comment:
I think if the file doesn't exist, just download it from github (with urllib.request.urlopen into a tmpdir and ZipFile.extract).  Also add them to `examples/.gitignore` (which doesn't exist right now).  This way the example will work both on CI and locally with no extra work from the end user.